src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -308,10 +309,6 @@ def map_to_bools(inputs: Any) -> Any:
     np.log2: sp_func._log2,
     np.log10: sp_func._log10,
     np.log1p: lambda col: sp_func.ln(col + 1),
-    # Aggregate functions translate to identity functions when applied element wise
```

Comment:
Is this removed because we added the block above for handling aggregate functions?

Reply:
Right.